opal/templates/patient_lists/spreadsheet_list.html

```diff
@@ -220,6 +221,14 @@ <h4 class="print-only">
 
 {% block footer %}
 
+{% for column in columns %}
+  {% if column.model %}
+    {% for prefix in column.prefixes %}
```

Comment:
Does that mean I include it multiple times if there are prefixes ?

Not sure why we need to loop over this if we then pass it in as an argument ?